core/src/main/java/com/ericsson/bss/cassandra/ecchronos/core/repair/DefaultRepairConfigurationProvider.java

```diff
@@ -72,7 +78,7 @@ public void onKeyspaceChanged(KeyspaceMetadata current, KeyspaceMetadata previou
     @Override
     public void onTableAdded(TableMetadata table)
     {
-        if (myReplicatedTableProvider.accept(table.getKeyspace().getName()))
+        if (myReplicatedTableProvider.accept(table.getKeyspace().getName()) && !isTableIgnored(table))
```

Comment:
Would this logic not be better suited in ReplicatedTableProvider? We already have logic which ignores tables in ReplicatedTableProvider.accept() and it would be nice to have all ignore logic in the same place.

Reply:
I don't think so, replicatedTableProvider.accept is used for both ondemand and schedules. This setting is global and can be overriden in schedule.yaml for the specific tables. I've moved the code around a bit so the validation is done in the same place as validation if it's disabled.